### PR TITLE
Properly consider <base> tags for HTML imports

### PR DESCRIPTION
**Pull request by drarmstr**
Discussed with @jmesserly for Issue #329 

I was added to the corporate-level Contributor License Agreement with Intel under douglas.armstrong@intel.com

### CODE REVIEW

**Comment by jmesserly** [comment]
LGTM. @sorvell want to take a look as well?

If it's not too much trouble, might be good to add a unit test as well, that way we don't accidentally regress it.

Thanks for putting this together!

**Comment by addyosmani** [comment]
LGTM2. Agree on us landing a test alongside this change. Think you might have time for that @drarmstr

**Comment by addyosmani** [comment]
friendly ping @drarmstr

**Comment by drarmstr** [comment]
I provided my local test case in #329.  I'm not too familiar with your test system how that would integrate.

**Comment by drarmstr** [comment]
Is this pull request still a candidate for merging?

**Comment by azakus** [comment]
This PR fails to complete test runs in firefox and safari.

File: src/HTMLImports/importer.js [hunk 1]
```diff
@@ -128,22 +128,28 @@ function makeDocument(resource, url) {
...
-  // establish a relative path via <base>
-  var base = doc.createElement('base');
-  base.setAttribute('href', url);
-  // add baseURI support to browsers (IE) that lack it.
-  if (!doc.baseURI && !hasBaseURIAccessor(doc)) {
-    // Use defineProperty since Safari throws an exception when using assignment.
-    Object.defineProperty(doc, 'baseURI', {value:url});
+  
+  // install html
+  doc.body.innerHTML = resource;
```
Comment (azakus) [inline src/HTMLImports/importer.js]: why is this reordered before setting the `<base>` tag?

Comment (drarmstr) [inline src/HTMLImports/importer.js]: My thinking was so the doc content would be set so we could search for any existing `<base>` tags with the `doc.querySelector('base')` line below.  Is there a concern with this?

**Comment by drarmstr** [comment]
@azakus Trying to keep this moving along I tried to investigate the travis-ci failure in case that is holding things up.  I setup a web-component-tester environment and tried to duplicate the tests.  I do see failures in the HTMLImports test on FireFox and Chrome.  However, I also see failures when I run with the unmodified webcomponentsjs master.  So, perhaps these are existing known issues?

**Comment by TimvdLippe** [comment]
This was fixed in https://github.com/webcomponents/html-imports/commit/2b10492d48b8d5fdd4187510ee6175e0f25967ce Sorry for the late response!

**Pull request closed without merging**